.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+	root: true,
+	env: {
+		browser: true,
+		es2021: true,
+		node: true,
+	},
+	parserOptions: {
+		project: './tsconfig.json',
+		tsconfigRootDir: __dirname,
+		ecmaVersion: 12,
```

Comment:
Line number 5 in this file: automatically sets the ecmaVersion parser option to 12, so this is duplicate.

If you want to keep both `es2021: true,` and `ecmaVersion: 12,` good to add comments so any future developer doesn't accidentally mis-configure